src/Model.cpp

```diff
@@ -54,6 +54,8 @@ Version History
 #include <mpi.h>
 #include <math.h>
 #include <string.h>
+#include <string>
```

Comment:
Does it need `<string.h>` AND `<string>`?

Reply:
Yes. string.h is a c class for character arrays. string is the cpp version for its string classes.